inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -340,26 +342,37 @@ void MKLDNNGraph::Replicate(const ICNNNetwork &network, const MKLDNNExtensionMan
 void MKLDNNGraph::InitGraph() {
     MKLDNNGraphOptimizer optimizer;
 
+    OV_ITT_TASK_CHAIN(taskChain, MKLDNNPlugin::itt::domains::MKLDNN_LT, "InitGraph", "InitNodes");
```

Comment:
If fact we cover each function call inside InitGraph. I suggest to clean this InitGraph function from use of ITT_TASK_CHAIN  and move counters inside step functions.
You will need to wrap some code into separate method:
1. Wrap it into `initOptimalPrimitiveDescriptors()`
```
for (auto &node : graphNodes) {
         OV_ITT_TASK_NEXT(taskChain, node->profiling.initOptimalPrimitiveDescriptor);
         node->initOptimalPrimitiveDescriptor();
     }
```
2. Wrap it into `executeConstantNodesOnly()`
```
 mkldnn::stream stream = mkldnn::stream(stream::kind::eager);
    for (auto &graphNode : graphNodes) {
        if (!graphNode->isConstant())
            continue;
        graphNode->execute(stream);
    }
```